### ISSUE TITLE
[FeatureHighlight] MDCFeatureHighlightViewController a11y hint "Double tap to dismiss" is inaccurate

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#4839]
From the internal bug:
> When MDCFeatureHighlightViewController appears, focus moves to "Tap to start a new meeting" text and voiceover announces "double tap to dismiss". But double tap is not dismissing the MDCFeatureHighlightViewController

---

This is an internal issue. If you are a Googler, please visit [b/117438991](http://b/117438991) for more details.

---
#### Internal data
- Associated internal bug: [b/117438991](http://b/117438991)

**Comment by material-mirror** [comment]
The internal issue is now closed, closing this issue.

**Comment by material-mirror** [comment]
The internal issue is now closed, closing this issue.

**Comment by romoore** [comment]
b/117438991 is not closed.

**Comment by material-mirror** [comment]
The internal issue [b/117438991](http://b/117438991) is now closed. This issue is being closed as a result.

**Comment by mparmar-covantex** [comment]
Issue is not closed, still facing same issue. @material-mirror , @material-admin
